sql/core/pom.xml

```diff
@@ -86,15 +86,17 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-storage-api</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.apache.orc</groupId>
       <artifactId>orc-core</artifactId>
-      <classifier>${orc.classifier}</classifier>
```

Comment:
Our native orc upgrade should be independent on Hive ORC reader, as @dongjoon-hyun pointed out.